.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npm install
```

Comment:
Should we make the same change on `publish.yml` and `feature.yml`?

Reply:
Oh, I didn't even realize there was another `.yml` file. That's probably why the Prettier workflow only failed after the PR landed.

The workflows have the same contents, so lemme just merge them.